template/php7/Dockerfile

```diff
@@ -0,0 +1,48 @@
+# start with the official Composer image and name it
+FROM composer:1.7 AS composer
+
+# continue with the official PHP image
+FROM php:7.2-alpine
+
+# copy the Composer PHAR from the Composer image into the PHP image
+COPY --from=composer /usr/bin/composer /usr/bin/composer
+
+# Composer requirements
+ARG COMPOSER_AUTH='{}'
+ENV COMPOSER_AUTH=${COMPOSER_AUTH}
+
+# Add watchdog and packages
+RUN apk add --no-cache git && \
+    echo "Pulling watchdog binary from Github." && \
+    curl -sSL https://github.com/openfaas/faas/releases/download/0.9.4/fwatchdog > /usr/bin/fwatchdog && \
+    chmod +x /usr/bin/fwatchdog
+
+# create non-root user
+RUN addgroup -S app && adduser -S -g app app && \
+    mkdir -p /home/app
+
+# Import function
+WORKDIR /home/app
+COPY index.php composer.* ./
```

Comment:
@andrew-s there won't be a any `composer.*` files here surely, just copy `index.php` here?

Reply:
Yeah, missed that, updated and submitted.